log_injection/irzaip_ChippyBot/wa.py

```
@@ -0,0 +1,478 @@
+    if user_number in conversations:
+        return {"messages" : str(conversations[user_number].messages)} # type: ignore
+    else:
+        logging.error(f"save_log error finding user_number: {str(user_number)}")
+
+@app.get("/save_logs")
+async def save_logs() -> dict[str, str] | None:
+    """save semua logs conversation ke dalam file log"""
+    for i in conversations.keys():
+        try:
+            save_log(i)
+            return {"save":"done"}
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)